### DISCUSSION TITLE
Minify Tailwind CSS Class names in Next.js

### DISCUSSION THREAD

**Discussion by surjithctly** [vercel/next.js#24092] [Help]
Is there any way to minify / obfuscate Tailwind CSS Class names in Next.js? What are the options?

Eg:

Input:
```jsx
<div className="flex space-x-3 sm:mb-4 text-sm font-medium hover:text-gray-500">
...
</div>
```
Output:
```jsx
<div className="qb Kx sm:yb nH wZ hover:Yk">
...
</div>
```

**Comment by gggglglglg** [comment]
I tried to use https://github.com/sndyuk/mangle-css-class-webpack-plugin , but it did not work properly for me
My problem is described here https://github.com/sndyuk/mangle-css-class-webpack-plugin/issues/46

Does anyone have any other ideas on how to make the tailwind obfuscation in Next.js ?
